SpatialGDK/Source/SpatialGDK/Private/Interop/GlobalStateManager.cpp

```diff
@@ -74,6 +75,32 @@ void UGlobalStateManager::ApplyDeploymentMapData(Schema_ComponentData* Data)
 	SchemaHash = Schema_GetUint32(ComponentObject, SpatialConstants::DEPLOYMENT_MAP_SCHEMA_HASH);
 }
 
+void UGlobalStateManager::ApplySnapshotVersionData(Schema_ComponentData* Data)
+{
+	Schema_Object* ComponentObject = Schema_GetComponentDataFields(Data);
+
+	SnapshotVersion = Schema_GetUint64(ComponentObject, SpatialConstants::SNAPSHOT_VERSION_NUMBER_ID);
+
+	if (NetDriver != nullptr && NetDriver->IsServer())
```

Comment:
As this gets called on both clients and server, any reason to not check both here?

Reply:
Oh I guess the query callback will be issues regardless - and we wouldn't want it to do any functionality :hmm: